torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19542,6 +19543,16 @@ def __init__(
         validate_view_consistency=False
     ),
     #
+    # torch.linalg
+    #
+    PythonRefInfo(
```

Comment:
why is regular vector_norm OpInfo not a ReductionInfo?

Reply:
I don't know. I'll try to make it into a `ReductionInfo`.

Reply:
Done in https://github.com/pytorch/pytorch/pull/78424. I thought I was going to need to fix the operation itself (hence the new PR), but the tests passed without having to modify it.